interface/billing/print_daysheet_report_num1.php

```diff
@@ -68,1764 +68,1846 @@
+                                $pro19_patref = $pro19_patref + $iter['pat_code'];
+                            }
+                            break;
                     }
 
-                    if (($iter['code_type']) != 'Insurance Payment' and ($iter['code_type']) != 'Patient Payment' and $iter['paytype'] != 'PCP') {
-                        print  "</span></td><td width='100'><span class='text'><center>" . text($iter['code_type']) . "</center>";
+                    if ($the_first_time === 1) {
+                        $user = $iter['user'];
+                        $new_old_pid = $iter['pid'];
+                        $the_first_time = 0;
                     }
 
-                    print  "</span></td><td width='100'><span class='text'>";
-                    print  "</span></td>\n";
-                } else {
-                    if ($iter['fee'] != 0) {
-                        $line_total = $line_total + $iter['fee'];
-                        print "<td width='70'><span class='text'><center>" . text(date("Y-m-d", strtotime($iter['date']))) . "</center>";
-                        print "</span></td><td width='50'><span class='text'><center>" . text($iter['pid']) . "</center>";
-                        print "</span></td><td width='180'><span class='text'><center>" . text($iter['last']) . ", " . text($iter['first']) . "</center>";
-                        if ($GLOBALS['language_default'] === 'English (Standard)') {
-                               print "</span></td><td width='100'><span class='text'><center>" . text(ucwords(strtolower(substr($iter['code_text'], 0, 25)))) . "</center>";
-                        } else {
-                            print "</span></td><td width='100'><span class='text'><center>" . text(substr($iter['code_text'], 0, 25)) . "</center>";
+                    if ($totals_only != 1) {
+                        if ($old_pid != $iter['pid'] and ($iter['code_type'] != 'payment_info')) {
+                            if ($old_pid === $new_old_pid) {
+                                $line_total = 0;
+                                $line_total_pay = 0;
+                            }
+
+                            if ($first_time) {
+                                print "<div class='table-responsive'><table class='table'>";     // small table
+                                $first_time = 0;
+                            } ?>
+
+                            <!-- TODO: Further replace with classes -->
+                            <tr>
+                                <td width='70'>
+                                    <span class='text text-center font-weight-bold'><?php echo xlt("Date"); ?></span>
+                                </td>
+                                <td width='50'>
+                                    <span class='text text-center font-weight-bold'><?php echo xlt("Acct"); ?>#</span>
+                                </td>
```

Comment:
Hey @tywrenn 

I convert the print function to pure HTML and remove some invalid code, just wanna ask you, should I remove `span` tag and move the class to `td` tag?

Like this:
```html
<td width='70' class='text text-center font-weight-bold'>
    <?php echo xlt("Date"); ?>
</td>
```

Reply:
@stu1509 yeah u can do that